firstvoices/backend/serializers/site_serializers.py

```diff
@@ -115,6 +121,34 @@ class Meta(SiteSummarySerializer.Meta):
         )
 
 
+class SiteDetailWriteSerializer(SiteDetailSerializer):
+    logo = serializers.SlugRelatedField(
```

Comment:
You could consider switching SlugRelatedField with slug_field="id" to just StringRelatedField, which automatically uses the id

Reply:
Yes, thank you for the suggestion. The StringRelatedField is read only, but the PrimaryKeyRelatedField worked great. I've changed all 3 fields to using PrimaryKeyRelatedField now.